### DISCUSSION TITLE
Заблокировать выбор даты идущей через 10 дней за сегодняшней

### DISCUSSION THREAD

**Discussion by yrtchs** [t1m0n/air-datepicker#643] [Announcements]
Т.е. как пример - сегодня 1 число и с 1 по 10 выбор чисел активен. А на 11+ заблокировать.

**Comment by nicothin** [comment]
Почитать документацию про maxDate )))
Как-то так, к примеру:
```
  var now = +new Date();
  var endDate = new Date(now + 1000 * 60 * 60 * 24 * 10);
  $( '#modal-datepicker' ).datepicker({
    maxDate: endDate,
  });
```

**Comment by zava75** [comment]
> Почитать документацию про maxDate))) 
> Как-то так, к примеру:
> 
> ```
>   var now = +new Date();
>   var endDate = new Date(now + 1000 * 60 * 60 * 24 * 10);
>   $( '#modal-datepicker' ).datepicker({
>     maxDate: endDate,
>   });
> ```

я так понял это самое короткое решение `maxDate: new Date(+new Date + 1000 * 60 * 60 * 24 * 60), // max date 2 month` , нету решений типа `  dateFormat: "yy-mm-dd",
  maxDate: "+2m"`?

**Comment by bladeaweb** [comment]
А можно вопрос зачем такие извращения с умножениями и прочим?
https://codepen.io/blade_didan/pen/OemmjY
```
var currentDate = new Date();
var someDate = new Date();
someDate.setMonth(someDate.getMonth() + 1);
$('.date').datepicker({
  minDate: currentDate,
  maxDate: someDate,
});
```
Вот пример ограничения минимума и максимума на сегодня как минимум и на месяц вперед как максимум, это же не сложно, при желании можно минимальную дату задавать какую угодно, так же можно и максимальную менять если надо то можно хоть несколькими днями, хоть несколькими годами ограничивать, на все ваша фантазия.
На всякий случай вот вам пример на +10 дней:
```
var currentDate = new Date();
var someDate = new Date();
someDate.setDate(someDate.getDate() + 10);
$('.date').datepicker({
  minDate: currentDate,
  maxDate: someDate,
});
```
